src/app/shared/dovetail/dovetail-audio-event.ts

```diff
@@ -21,3 +21,13 @@ export class DovetailAudioEvent {
     return event;
   }
 }
+
+function createEvent(name):Event {
+  try {
+    return new Event(name);
+  } catch (e) {
+    let event:any = document.createEvent('CustomEvent');
+    event.initCustomEvent(name, false, false, undefined);
```

Comment:
you would *think* this would be equivalent to `initCustomEvent(name, false, false)`, but then, you've never worked on the IE team, so...